### ISSUE TITLE
Maven artifacts missing

### ISSUE THREAD

**Issue by ms32035** [djspiewak/sbt-github-packages#39]
Trying to use this in a GH action, and looks like the maven artifacts are missing. Can't find the plugin through maven central either, and the links refer to bintray, which will soon be decommissioned. 
```
[warn] 
[warn] 	Note: Some unresolved dependencies have extra attributes.  Check that these dependencies exist with the requested attributes.
[warn] 		com.codecommit:sbt-github-packages:0.5.2 (sbtVersion=1.0, scalaVersion=2.12)
[warn] 
[warn] 	Note: Unresolved dependencies path:
[error] sbt.librarymanagement.ResolveException: Error downloading com.codecommit:sbt-github-packages;sbtVersion=1.0;scalaVersion=2.12:0.5.2
[error]   Not found
[error]   Not found
[error]   not found: https://repo1.maven.org/maven2/com/codecommit/sbt-github-packages_2.12_1.0/0.5.2/sbt-github-packages-0.5.2.pom
[error]   not found: /home/runner/.ivy2/localcom.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml
[error]   download error: Caught java.io.IOException: Server returned HTTP response code: 403 for URL: https://dl.bintray.com/sbt/sbt-plugin-releases/com.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml (Server returned HTTP response code: 403 for URL: https://dl.bintray.com/sbt/sbt-plugin-releases/com.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml) while downloading https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/com.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml
[error]   download error: Caught java.io.IOException: Server returned HTTP response code: 403 for URL: https://dl.bintray.com/typesafe/ivy-releases/com.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml (Server returned HTTP response code: 403 for URL: https://dl.bintray.com/typesafe/ivy-releases/com.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml) while downloading https://repo.typesafe.com/typesafe/ivy-releases/com.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml
[error] 	at lmcoursier.CoursierDependencyResolution.unresolvedWarningOrThrow(CoursierDependencyResolution.scala:258)
[error] 	at lmcoursier.CoursierDependencyResolution.$anonfun$update$38(CoursierDependencyResolution.scala:227)
[error] 	at scala.util.Either$LeftProjection.map(Either.scala:573)
[error] 	at lmcoursier.CoursierDependencyResolution.update(CoursierDependencyResolution.scala:227)
[error] 	at sbt.librarymanagement.DependencyResolution.update(DependencyResolution.scala:60)
[error] 	at sbt.internal.LibraryManagement$.resolve$1(LibraryManagement.scala:53)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$12(LibraryManagement.scala:103)
[error] 	at sbt.util.Tracked$.$anonfun$lastOutput$1(Tracked.scala:73)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$20(LibraryManagement.scala:116)
[error] 	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11(LibraryManagement.scala:116)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11$adapted(LibraryManagement.scala:97)
[error] 	at sbt.util.Tracked$.$anonfun$inputChangedW$1(Tracked.scala:219)
[error] 	at sbt.internal.LibraryManagement$.cachedUpdate(LibraryManagement.scala:130)
[error] 	at sbt.Classpaths$.$anonfun$updateTask0$5(Defaults.scala:3486)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error] 	at sbt.Execute.work(Execute.scala:291)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[error] 	at java.base/java.lang.Thread.run(Thread.java:834)
[error] (update) sbt.librarymanagement.ResolveException: Error downloading com.codecommit:sbt-github-packages;sbtVersion=1.0;scalaVersion=2.12:0.5.2
[error]   Not found
[error]   Not found
[error]   not found: https://repo1.maven.org/maven2/com/codecommit/sbt-github-packages_2.12_1.0/0.5.2/sbt-github-packages-0.5.2.pom
[error]   not found: /home/runner/.ivy2/localcom.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml
[error]   download error: Caught java.io.IOException: Server returned HTTP response code: 403 for URL: https://dl.bintray.com/sbt/sbt-plugin-releases/com.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml (Server returned HTTP response code: 403 for URL: https://dl.bintray.com/sbt/sbt-plugin-releases/com.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml) while downloading https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/com.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml
[error]   download error: Caught java.io.IOException: Server returned HTTP response code: 403 for URL: https://dl.bintray.com/typesafe/ivy-releases/com.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml (Server returned HTTP response code: 403 for URL: https://dl.bintray.com/typesafe/ivy-releases/com.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml) while downloading https://repo.typesafe.com/typesafe/ivy-releases/com.codecommit/sbt-github-packages/scala_2.12/sbt_1.0/0.5.2/ivys/ivy.xml
[warn] Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? (default: r)
```

**Comment by csar** [comment]
Same here and for other sbt plugins that are hosted on bintray - bintray was supposed to shutdown May 1st...

**Comment by zackangelo** [comment]
Came here to say the same thing. Ironically, we're trying to migrate off of Bintray using this sbt plugin and can't get access to the existing artifacts. 🙃

**Comment by ms32035** [comment]
@djspiewak this was caused by bintray brown-out before decommissioning (https://status.bintray.com/). Can you republish elsewhere?

**Comment by djspiewak** [comment]
I'll republish to maven central this week.

**Comment by djspiewak** [comment]
Published 0.5.3 to Maven Central
